frontend-shared/README.md

```diff
@@ -0,0 +1,26 @@
+# Hypothesis frontend shared
+
+A place for shared frontend components, styling, and resources among Hypothesis frontend apps.
```

Comment:
This is somewhat future-aspirational. Perhaps simplifying to  "resources" for now? e.g.

`A package of resources for Hypothesis front-end applications.`

In fact, this sentence maybe isn't necessary at all, other than it being explicit that there is a package involved...up to you!